src/TabViewPanResponder.js

```diff
@@ -41,8 +41,6 @@ function forHorizontal(props: Props) {
       return gestureState.vx < 0;
     } else if (startDirection < 0) {
       return gestureState.vx > 0;
-    } else {
```

Comment:
@julienvincent But it's still returning a falsy value, so nothing will actually change here, except you get eslint error.